doc/howto/storage_pools.md

```diff
@@ -183,7 +183,22 @@ Therefore, you must first create a pending storage pool on each member with the
 Make sure to use the same storage pool name for all members.
 Then create the storage pool without specifying the `--target` flag to actually set it up.
 
-For example, the following series of commands sets up a storage pool with the name `my-pool` at different locations and with different sizes on three cluster members:
+Also see {ref}`cluster-config-storage`.
+
+```{note}
+For most storage drivers, the storage pools exist locally on each cluster member.
+That means that if you create a storage volume in a storage pool on one member, it will not be available on other cluster members.
+
+This behavior is different for Ceph-based storage pools (`ceph`, `cephfs` and `cephobject`) where each storage pool exists in one central location and therefore, all cluster members access the same storage pool with the same storage volumes.
+```
+
+### Examples
+
+See the following examples for how to create a storage pool in a cluster using different storage drivers.
```

Comment:
```suggestion
See the following examples for different storage drivers for instructions on how to create local or remote storage pools in a cluster.
```

Reply:
(Since the focus here is not to give examples for all storage drivers, but to give some examples for local and remote.)

Reply:
Fixed.